doc_swift_ui_gallery/swift_ui_gallery/swift_ui_gallery/primitives/image/localization/ImageLocalizationPage.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftUI
+
+struct ImageLocalizationPage: View {
+    @State private var localize: Bool
```

Comment:
Maybe I'm missing something, but I don't see where this toggle value is being used?

Reply:
it is used at line 28, to toggle between english and italian locales